apps/src/sites/hourofcode.com/pages/views/hoc_events_map_replacement.js

```diff
@@ -1,4 +1,4 @@
-let mapboxgl;
+/*global mapboxgl*/
```

Comment:
What is the purpose of "global"?  Is it used for linters?

Reply:
What scoping issue did you fix?

Reply:
Yep, `global` is used to satisfy linters. I was having an issue where `mapboxgl` was undefined despite the script that initialized it being loaded. I think it was because I had this let statement here.